src/PKSimRDependencyResolution/filelist.txt

```diff
@@ -0,0 +1,62 @@
+Autofac.pdb
```

Comment:
I don't get waht this file is? Why do we need this?
And can we generate it automatically instead of statically?

Reply:
why are we copying all xmls?

Reply:
why are we copying all pdbs?

Reply:
It is the opposite in fact, this file contains all the files that we want to EXCLUDE, this is the way we achieve that:
- 7z a pk-sim-r-dependencies.zip -x@filelist.txt * --> "-x" does the trick.
The thing is, we need to exclude certain files, but we do not know in advance which ones, given that we need some .xms for example.
So the approach here is that if we were to include some dependencies in the future, and forgot to update this file, the worst case scenario will be having a zip with more files than we need, and not otherwise, which could cause not to have the files needed for it to run properly